.changeset/hungry-ads-cheer.md

```diff
@@ -0,0 +1,13 @@
+---
+'braid-design-system': patch
+---
+
+---
+updated:
+  - Text
+  - Heading
+---
+
+**Text, Heading:** Ensure `maxLines` truncates correctly in flex container
+
+Provide `min-width` to `Text` and `Heading` components to ensure `maxLines` truncates as expected when typography components are used as a direct child of a flex container.
```

Comment:
Feel like this is implementation detail. Could honestly confuse users into thinking they need to make a change.